node/core/dispute-coordinator/src/metrics.rs

```diff
@@ -147,6 +155,14 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
+			refrained_participations: prometheus::register(
+			prometheus::Counter::with_opts(
+				prometheus::Opts::new(
+					"polkadot_parachain_dispute_refrained_participations",
+					"Number of refrained participations.",
```

Comment:
It would be nice to have a more detailed description of what the metric means, like below:
```suggestion
					"Number of refrained participations. We refrain from participation if all of the following conditions are met: disputed candidate is not included, not backed and not confirmed.",
```

Reply:
This is really better, thanks for the suggestion!